.github/workflows/build.yaml

```diff
@@ -0,0 +1,25 @@
+name: Container build & push
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [main]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build-image:
+    runs-on: ubuntu-20.04
```

Comment:
Any reason why one action is using `ubuntu-latest` and the other `ubuntu-20.04` 😃 ?